main.tf

```diff
@@ -1,13 +1,18 @@
+provider "azurerm" {
+  version = ">=1.40.0"
```

Comment:
This will cause problems if we're using a different provider version

Reply:
I can knock it down to a lesser version, but I haven't tested it on those.